### PR TITLE
Complete the document content

### PR DESCRIPTION
**Pull request by miaoyq**
Signed-off-by: Yanqiang Miao <miao.yanqiang@zte.com.cn>

### CODE REVIEW

File: docs/reference/commandline/plugin_create.md [hunk 1]
```diff
@@ -41,8 +41,9 @@ $ ls -ls /home/pluginDir
 $ docker plugin create plugin /home/pluginDir
 plugin
 
-NAME                  	TAG                 DESCRIPTION                  ENABLED
-plugin                  latest              A sample plugin for Docker   true
+$ docker plugin ls
+ID                  NAME                TAG                 DESCRIPTION                  ENABLED
+672d8144ec02        plugin              latest              A sample plugin for Docker   true
```
Comment (cpuguy83) [inline docs/reference/commandline/plugin_create.md]: Only thing is, `Enabled` would be false, here.... otherwise LGTM

**Review by thaJeztah — COMMENTED** [review]
LGTM after @cpuguy83's comment is addressed

**Comment by miaoyq** [comment]
@cpuguy83 @thaJeztah 
updated, PTAL Thanks

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**